src/test/cypress/e2e/support/index.js

```diff
@@ -5,3 +5,23 @@ import './app'
 import './app-frontend'
 import 'cypress-file-upload';
 import 'cypress-plugin-tab'
+import './start-app-instance'
+
+before(() => {
+  if (Cypress.env('environment') != "local") {
```

Comment:
Does this actually do anything? I'm guessing the app would work fine even though this call is never mocked?